proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -87,6 +87,25 @@ message MonthlyUsage {
   uint32 total_method_types = 5;
 }
 
+// RepositoryUsage contains the total amount of message, enum and method types
+// for an repository within an organization.
```

Comment:
```suggestion
// for an repository.
```
there is no grouping by owner etc, and its not for org only, so this seems confusing